public/index.html

```diff
@@ -203,7 +203,7 @@ <h2 class="mb-5">Locale</h2>
         const client = new Appwrite.Client();
 
         client
-            .setEndpoint("http://localhost/v1") // Set your own appwrite server endpoint here, if not sure, you can get this value from your project settings page.
+            .setEndpoint("https://cloud.appwrite.io/v1") // Set your own appwrite server endpoint here, if not sure, you can get this value from your project settings page.
```

Comment:
Let's revert the URL ?

Reply:
It was actually intentional. Cloud is easier to quick start so we decided to keep this setter for better visibility of self hosting, but using Cloud as default.

Reply:
Okay 👍